docs/get-ckb.md

```diff
@@ -36,7 +36,7 @@ You also need to get the following packages：
 #### Ubuntu and Debian
 
 ```shell
```

Comment:
Could u help to update this language keyword, please?
Since [`shell` is not a keyword for highlight](https://github.com/github/linguist/blob/2734ca0134c0afe3c62ecb1ec8c721a28c8b9e63/lib/linguist/languages.yml#L4626-L4633), and the previous code block use `bash`.

Reply:
Maybe we should do that in another PR.